docs/workforce-management-framework/topics/ref-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,70 @@
+[id='ref-introducing-raincatcher-{chapter}']
+= {WFM-RC-NameShort} Reference Material
+
+This section includes:
+
+. xref:highlighting-raincatcher-areas-of-focus-{chapter}[Highlighting {WFM-RC-NameShort} Areas of Focus]
+. xref:describing-raincatcher-technologies-and-technicalities-{chapter}[Describing {WFM-RC-NameShort} Technologies and Technicalities]
+. xref:identifying-raincatcher-repositories-{chapter}[Identifying {WFM-RC-NameShort} Repositories]
+. xref:highlighting-raincatcher-tech-preview-differences-{chapter}[Highlighting {WFM-RC-NameShort} and 'Tech Preview' Differences]
+
+[id='highlighting-raincatcher-areas-of-focus-{chapter}']
+== Highlighting {WFM-RC-NameShort} Areas of Focus
+
+.Areas of Focus
+|===
+|Area |Focus
+
+|Framework
+|A set of modules for building mobile and website Workforce Management solutions
+
+|Modules
```

Comment:
This section doesn't really do anything for the reader - can we elaborate more here?